### DISCUSSION TITLE
Unable to filter on COMM with a package name.

### DISCUSSION THREAD

**Discussion by x1mus** [yanivagman/BPFroid#18] [Q&A] [answered]
Hello, I am trying to use a command such as:
`./tracee --trace comm='my.super.package.name'`

I realized that this does not work (because of the dots ?). I tried escaping them, just placing a partial filter but nothing works.

I'm guessing I should be updating something in `tracee.go` to enable such a feature but cant figure out how.

**Comment by yanivagman** [comment, marked as answer]
Hello,
Package filters should be given using `package` and not `comm`
See here: https://github.com/yanivagman/BPFroid/blob/main/main.go#L466
